core/src/main/java/org/frankframework/management/bus/endpoints/ConfigManagement.java

```diff
@@ -104,11 +104,11 @@ public Message<String> getConfigurationDetailsByName(Message<?> message) {
 			return new JsonMessage(Collections.singletonList(new ConfigurationDTO(configuration)));
 		}
 
-		List<ConfigurationDTO> configs = new LinkedList<>();
-		for (Configuration configuration : getIbisManager().getConfigurations()) {
-			configs.add(new ConfigurationDTO(configuration));
-		}
-		configs.sort(new ConfigurationDTO.NameComparator());
+		List<ConfigurationDTO> configs = getIbisManager().getConfigurations()
+				.stream()
+				.map(ConfigurationDTO::new)
+				.sorted(new ConfigurationDTO.NameComparator())
+				.toList();
```

Comment:
Let op dat 'toList' een immutable list oplevert, waar dit voorheen een wel te bewerken lijst was. Mogelijk wil je hier `.collect(Collectors.toList())` doen

Reply:
Het maakt toch niet uit dat hij immutable is?

Reply:
Ik had het uitgezocht en omdat deze verder niet meer gemuteerd wordt, had ik gekozen voor `.toList()` ipv `.collect(Collectors.toList())`.

Reply:
Prima, als het maar een bewuste keuze is!